packages/fluentui/docs/src/examples/components/Input/index.tsx

```diff
@@ -4,13 +4,15 @@ import Rtl from './Rtl';
 import Types from './Types';
 import Variations from './Variations';
 import State from './State';
+import Slots from './Slots';
 
 const InputExamples = () => (
   <div>
     <Types />
     <Variations />
     <State />
     <Rtl />
+    <Slots />
```

Comment:
I suggest to move it before `RTL` section

Reply:
Done